regoservice/eval.go

```diff
@@ -183,15 +183,47 @@ func BuildBundledRegoEvaluator(rego_package string) (data.Inpteval, error) {
 		aggrQuery:        aggrQuery,
 	}, nil
 }
+
+// loadsFuncs try to load rego files from the filesystem,
```

Comment:
it builds collection of function - nothing is loaded

Reply:
`return []func(r *rego.Rego){rego.Load(foundPaths, nil)}`
from the opa package:

> // Load returns an argument that adds a filesystem path to **load** data
// and Rego modules from.

Reply:
You are correct but actual loading happens outside of this method call (when returned function is invoked). It's not a big issue anyway, we can improve this eventually